app/views/carts/_cart.html.erb

```diff
@@ -0,0 +1,9 @@
+<div id="<%= dom_id cart %>">
+  <h2 class="font-bold text-lg mb-3">Your Pragmatic Cart</h2>
+
+  <ul class="list-disc list-inside">    
+    <% cart.line_items.each do |item| %>
+      <li><%= item.product.title %></li>
```

Comment:
It would be preferred to display line_items with the corresponding number. So, `x 1`, `x 2`, etc.

Reply:
✅